File: exercicios/Python/1013/1013.py
linha = input().split();

A = int(linha[0]);

B = int(linha[1]);

C = int(linha[2]);

MaiorAB = ((A + B) + abs(A - B ))/2;

MaiorABC = ((MaiorAB + C) + abs(MaiorAB - C))/2

print(round(MaiorABC), 'eh o maior');

File: exercicios/JavaScript/1013/1013.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var line = input.split(" ");

var a = parseInt(line.shift());
var b = parseInt(line.shift());
var c = parseInt(line.shift());

var maiorAB = (a+b + Math.abs(a-b)) / 2;

var maiorABC =  (c+maiorAB + Math.abs(maiorAB-c)) / 2;

console.log(maiorABC + ' eh o maior')
